crates/matrix-sdk-crypto/src/olm/signing/mod.rs

```diff
@@ -643,7 +643,7 @@ impl PrivateCrossSigningIdentity {
 
     /// Get the upload request that is needed to share the public keys of this
     /// identity.
-    pub(crate) async fn as_upload_request(&self) -> UploadSigningKeysRequest {
+    pub async fn as_upload_request(&self) -> UploadSigningKeysRequest {
```

Comment:
Hmm, why does this need to be pub? I don't think you ever get a hold of this type unless you're implementing a cryptostore but still, is this needed?

Reply:
This is used in the tests, I thought I had some compilation issue without it being exposed, but now it actually works fine so must have confused it